docs/1-Using-Fleet/3-REST-API.md

```diff
@@ -4005,3 +4104,274 @@ Retrieves the specified carve block. This endpoint retrieves the data that was c
 ```
 
 ---
+
+## Teams
+
+### Create team
+
+`POST /api/v1/fleet/teams`
+
+#### Parameters
+
+| Name       | Type    | In   | Description                                      |
+| ---------- | ------- | ---- | ------------------------------------------------ |
+| name   | string  | body | **Required.** The team's name.            |
```

Comment:
@noahtalerman Should we have a `description` param as well? I modeled that as I assumed we would probably want it.

Reply:
No, we don't need a `description` param for the team model. The current (planned) UI and `fleetctl` don't consume a `description` param and I can't think of a near-future use case in which we would want it.